### PR TITLE
Lifted mount id limit

### PR DESCRIPTION
**Pull request by EPuncker**
### Pull Request Prelude

- [x] I have followed [proper The Forgotten Server code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
- Lifted mount id limit and storage key ranges for outfits

**Issues addressed:** 
Mount id limit was about to be reached

code by @joseluis2g 

[cont]: https://github.com/otland/forgottenserver/wiki/Contributing
[code]: https://github.com/otland/forgottenserver/wiki/TFS-Coding-Style-Guide

### CODE REVIEW

**Comment by omarcopires** [comment]
Since #4197 has already been approved, wouldn't it be more interesting to merge it instead of changing the structure first?

**Comment by EPuncker** [comment]
> Since #4197 has already been approved, wouldn't it be more interesting to merge it instead of changing the structure first?

🤫

File: src/mounts.cpp [hunk 1]
```diff
@@ -25,8 +25,8 @@ bool Mounts::loadFromXml()
 
 	for (auto mountNode : doc.child("mounts").children()) {
 		uint16_t nodeId = pugi::cast<uint16_t>(mountNode.attribute("id").value());
-		if (nodeId == 0 || nodeId > std::numeric_limits<uint8_t>::max()) {
-			std::cout << "[Notice - Mounts::loadFromXml] Mount id \"" << nodeId << "\" is not within 1 and 255 range"
+		if (nodeId == 0) { //} || nodeId > std::numeric_limits<uint8_t>::max()) {
```
Comment (ranisalt) [inline src/mounts.cpp:28]: Why remove the check rather than just fixing the type?

File: src/const.h [hunk 2]
```diff
@@ -686,16 +686,16 @@ static constexpr int32_t CHANNEL_PARTY = 0x01;
...
 static constexpr int32_t PSTRG_OUTFITS_RANGE_START = (PSTRG_RESERVED_RANGE_START + 1000);
-static constexpr int32_t PSTRG_OUTFITS_RANGE_SIZE = 500;
-//[2001 - 2011];
+static constexpr int32_t PSTRG_OUTFITS_RANGE_SIZE = 1000;
+// [2001 - 2101];
 static constexpr int32_t PSTRG_MOUNTS_RANGE_START = (PSTRG_RESERVED_RANGE_START + 2001);
-static constexpr int32_t PSTRG_MOUNTS_RANGE_SIZE = 10;
-static constexpr int32_t PSTRG_MOUNTS_CURRENTMOUNT = (PSTRG_MOUNTS_RANGE_START + 10);
+static constexpr int32_t PSTRG_MOUNTS_RANGE_SIZE = 100;
+static constexpr int32_t PSTRG_MOUNTS_CURRENTMOUNT = (PSTRG_MOUNTS_RANGE_START + 999);
```
Comment (ranisalt) [inline src/const.h:698]: Why 999 if the range size is 100? And why 100 and not the actual limit?

Comment (joseluis2g) [reply src/const.h:698]: my bad there

**Pull request closed without merging**